app/views/users/show.html.erb

```diff
@@ -1,2 +1,22 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<div class="m-3 mx-auto" style="width: 32rem;">
+    <%= render 'shared/user', user: @user %>
+  <div class="card mb-3">
+    <p class="card-title">Bio:</p> <br> <br>
+    <p class="card-body"><%= @user.bio%></p>
+  </div>
+
+
+<% if @user.posts? %>
+    <% @user.three_most_recents_posts.each do |post| %>
+            <a href="/users/<%= @user.id %>/posts/<%= post.id %>" class="card text-decoration-none mb-3">
+                <div><%= render 'shared/post', post: post %></div>
+            </a>
```

Comment:
- [ ] Please let's ensure here that when a user clicks on a post, he can see that post and its comments. :ok_hand: 
```suggestion
            <a href="/users/<%= post.author.id %>/posts/<%= post.id %>" class="card text-decoration-none mb-3">
                <div><%= render 'shared/post', post: post %></div>
            </a>
```

Reply:
# Likewise here:
## Clicking on first post 
![image](https://github.com/vangelif/blog_app/assets/119002288/26242e80-51d6-4d7e-af30-bef4d48de498)
## Displaying of post and its comments
![image](https://github.com/vangelif/blog_app/assets/119002288/52c98aa1-2a0b-4a96-b975-e93a919c56a4)